tasks/config.yml

```diff
@@ -83,6 +83,18 @@
     dest: /usr/share/bbb-web/WEB-INF/classes/spring/turn-stun-servers.xml
   notify: restart bigbluebutton
 
+- name: create ssl folder for nginx
+  file:
+    path: /etc/nginx/ssl
+    state: directory
+
+
+- name: Generate DH Parameters with a different size (4096 bits)
```

Comment:
Could you make this step optional but default enabled (since it takes quite a while) and check, if the DH-params exist before creating them?

Reply:
That is also the reason, why the CI is failing. Because generating this exceeds the 10min timeout from travis...

Reply:
Thanks, looking good!